app/src/features/projects/components/ProjectFundingItemForm.tsx

```diff
@@ -53,6 +53,7 @@ export const ProjectFundingFormArrayItemYupSchema = yup.object().shape({
     .transform((value) => (isNaN(value) && null) || value)
     .typeError('Must be a number')
     .min(0, 'Must be a positive number')
+    .max(9999999999)
```

Comment:
this makes the error message that shows up say:
```
funding_amount must be less than.....
```

would it somehow be possible to change the message to say:
```
Funding amount must be less than.....
```

maybe by doing:
```
.max(9999999999, 'Funding amount must be less than...')
```

Reply:
"Funding amount must be less than $9,999,999,999"

Reply:
updated